BasicAPI6/LinkedTables.java

```diff
@@ -0,0 +1,61 @@
+package BasicAPI6;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.TreeMap;
+
+// класс для работы с таблицами связанными с экземплярами Notebook
+public class LinkedTables {
```

Comment:
Why do you need this class? Seems it does nothing

Reply:
This class is used to link IDs kept in Notebook objects with their descriptions kept in tables. For example, a Notebook object contains color ID, brand ID, model ID, etc. All additional data (the relevant color name, brand name, model name) are kept in linked tables and can be accessed by the relevant Notebook key and ID through the fields and methods of LinkedTables.